lib/schema/tablecompiler.js

```diff
@@ -100,7 +100,11 @@ class TableCompiler {
         : this._indexCommand('foreign', this.tableNameRaw, foreignData.column);
       const column = this.formatter.columnize(foreignData.column);
       const references = this.formatter.columnize(foreignData.references);
-      const inTable = this.formatter.wrap(foreignData.inTable);
+      const inTable = this.formatter.wrap(
+        foreignData.schema
+          ? `${foreignData.schema}.${foreignData.inTable}`
```

Comment:
@kibertoad is it ok to do this string concatenation (security-wise) I looked at the `wrap` function and it looks ok, just wanna double check